services/src/app.js

```diff
@@ -128,4 +128,9 @@ app.configure(jwt({
   }
 }));
 
+setInterval(() => {
```

Comment:
Couldn't we use the `cron` service, instead of manually using `setInterval`?

Reply:
This is in the services which don't have the cron package built in, I'm not going to add that dependncy when setInterval is built in already for just this one function :smile: